src/redux/books/books.js

```diff
@@ -0,0 +1,19 @@
+/* import { createStore } from 'redux';
```

Comment:
- [ ] Why this file has commented code only?

Reply:
Sorry, I started working on the redux but then I realized it was a duty for the next milestone